capella2polarion/connectors/polarion_worker.py

```diff
@@ -420,10 +414,35 @@ def _get_link_id(link: polarion_api.WorkItemLink) -> str:
             )
         )
 
-    def patch_work_items(
+    def compare_and_update_work_items(
         self, converter_session: data_session.ConverterSession
     ) -> None:
         """Update work items in a Polarion project."""
         for uuid, data in converter_session.items():
             if uuid in self.polarion_data_repo and data.work_item is not None:
-                self.patch_work_item(data)
+                self.compare_and_update_work_item(data)
+
+    def post_document(self, document: polarion_api.Document):
+        """Create a new Document."""
+        self.client.project_client.documents.create(document)
+
+    def update_document(self, document: polarion_api.Document):
+        """Create a new Document."""
+        self.client.project_client.documents.update(document)
+
+    def get_document(
+        self, space: str, name: str
+    ) -> polarion_api.Document | None:
+        """Create a new Document."""
```

Comment:
The docstrings are all the same? We update and get a document, not create it, or?

Reply:
Very true - typical copy&paste issue. Will fix